item_content/item.rb

```diff
@@ -0,0 +1,28 @@
+require 'date'
+class Item
+  attr_accessor :genre, :author, :source, :label, :publish_date
```

Comment:
- [ ] As stated in the project requirement, there needs to be a test file for all classes and their public methods. Currently, there is no test file for the `Item` class. I'll suggest you add a test for this class. :+1: :smile: